server.js

```diff
@@ -1,15 +1,62 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const cors = require('cors');
+// const requestPromise = require('request-promise');
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 7070;
+
+// bundles for each react component
+// const bundles = require('../public/services')
```

Comment:
You should take out dead code in this file.

Reply:
Removed!